spec/features/state_file/review_page_spec.rb

```diff
@@ -6,6 +6,10 @@
 
   before do
     allow_any_instance_of(Routes::StateFileDomain).to receive(:matches?).and_return(true)
+
+    # Turn on 1099-R support to be more like prod
+    allow(Flipper).to receive(:enabled?).and_call_original
+    allow(Flipper).to receive(:enabled?).with(:show_retirement_ui).and_return(true)
```

Comment:
the rest of the specs appear to do this -- this spec is only the income review & editing directly from that income review to the 1099R edit (which was outside the `show_retirement_ui` scope -- i think `show_retirement_ui` is more about the followups?

Reply:
You're right, false alarm on the flipper flag not hiding the right stuff. But I do still think we should either turn the flag on for this test, or create an NJ feature test that includes 1099-R followups and a review page visit, because apparently there is no test that hits the NJ review page with the show_retirement_ui flag on (or it would have failed when I accidentally broke that page)